MeetupPlatformApi/appsettings.json

```diff
@@ -4,5 +4,9 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+
```

Comment:
Consider removing redundant newline

Reply:
we consulted and decided to leave an empty line